pkg/delete/delete.go

```diff
@@ -96,7 +101,7 @@ func deleteConfig(ctx context.Context, clients client.ClientSet, t string, entri
 		log.With(log.TypeAttr(t)).WarnContext(ctx, "Skipped deletion of %d Classic configuration(s) as API client was unavailable.", len(entries))
 	} else if t == "bucket" {
```

Comment:
Just curious, how will the solution in the end look? A map with each and every classic API and resource, where settings is the fallback instead of the switch?

Reply:
Would we then still need to handle automation differently?